### ISSUE TITLE
History of high pressure injection injury to hand

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10833]
**Preferred term label:**
History of high pressure injection injury to hand

**Synonyms**

**Definition (free text, please give PubMed ID)**
Injury to the skin on the hand or finger from a high pressure injection such as a grease gun, paint gun, or water gun

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK542210/ and https://www.orthobullets.com/hand/12104/high-pressure-injection-injuries

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001116
- label: History of high-pressure injection injury of the hand
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:31194367
- def: Medical history of the hand having come into contact with the nozzle of a high-pressure injecting system such as a paint gun or air compressor.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10833
- synonyms:
